shiftcommerce/reference-site
# Issue: Migrate tests to trendy

### Overview

Related GitHub issue: https://github.com/shiftcommerce/trendy-golf/issues/243

PR cleans up existing cypress specs by addressing the following requirements:

- [x] Ensure all current tests are fully stubbed
- [x] Share the same test state where we can to avoid re-navigating where possible

### QA

1. Check out branch - `issue/243_migrate_tests_to_trendy`

2. Run `yarn cy:open` (don't forget to boot up your server)

### Code Review

All files

### Dependencies Review

N/A

## ENV variables to be added

N/A
